### DISCUSSION TITLE
Health check route

### DISCUSSION THREAD

**Discussion by vittaliy79** [stoplightio/prism#1840] [Q&A]
Hi. What would be the default route in prism server always responding with Status 200 which can be used as the service Health check route?

Thank you!

**Comment by EdVinyard** [comment]
Prism has no paths besides those you define in your mocked/proxied API, so there isn't a direct way to conduct a health or readiness check of Prism _itself_ via HTTP.  
- In mock mode, any path you've defined with the desired response can be used as a health/readiness check.  
- In proxy mode, even paths you define will be a combined health/readiness check of both Prism and the upstream API server.

The discussion and workarounds in https://github.com/stoplightio/prism/issues/2047 may be helpful.
